surfer/tests/test_viz.py

```diff
@@ -1,18 +1,18 @@
-import numpy as np
 import os
 import os.path as op
 from os.path import join as pjoin
-import re
 import shutil
-import subprocess
-from nose.tools import assert_equal
-from numpy.testing import assert_raises, assert_array_equal
 from tempfile import mkdtemp, mktemp
+
+import imageio
```

Comment:
ok but I would still make it optional. It's stupid to loose users for a use
case that represents a fraction of them. I don't have experience with this
lib to tell you if it actually works smoothly for everyone.